services/net/folder-collection/FolderCollectionManager.cs

```diff
@@ -417,30 +417,30 @@ private async Task<bool> RunFilterAsync(API.Areas.Services.Models.Content.Conten
             return false;
         }
 
-        if (filter.Settings.DateOffset.HasValue && publishedOn?.Date < now.Date.AddDays(filter.Settings.DateOffset.Value))
+        if (filter.Settings.DateOffset.HasValue && publishedOn?.Date < now.Date.AddDays(filter.Settings.DateOffset.Value > 0 ? filter.Settings.DateOffset.Value * -1 : filter.Settings.DateOffset.Value))
```

Comment:
Correctly handle the offset values.

Reply:
We should clear it up in the UI too, but yes they can be positive or negative.  But it of course makes no sense to look for content in the future.